test/core/index.js

```diff
@@ -1,10 +0,0 @@
-describe('index', function() {
```

Comment:
This file wasn't doing anything as `src/core/index.js` wasn't redefining these things